src/components/IconButton/useIconButtonStyles.tsx

```diff
@@ -70,6 +69,34 @@ export const getGhostButtonStyles = (theme: Theme, variantColor: ButtonColorVari
   };
 };
 
+export const getSolidButtonStyles = (theme: Theme, variantColor: ButtonColorVariant) => {
```

Comment:
We used to import this from the `button/useButtonStyles.tsx` file, but I realized that these  2  should not be locked in together